### DISCUSSION TITLE
MSFS FBW A320 Refuel not working

### DISCUSSION THREAD

**Discussion by ElectiXs** [flybywiresim/aircraft#8072] [Help] [answered]
Hey, i got the problem that the Refueling on my A320 FBW EFB doesn't work.

It says "refuel (unavailable)"

I tried different airports / gates.

Neither in the experimental nor in the stable version it works.

Any idea how to fix this?

![212539e44a58d49173156477386152dc](https://github.com/flybywiresim/a32nx/assets/136236670/e5f12c50-8871-430c-b5e9-0d442d0f39b8)

**Comment by rotowe** [comment, marked as answer]
If you're using GSX, you'll probably have to decativate the GSX Synchronizations in the FBW EFB Settings/3rd Party Options.
